templates/user/auth/activate.tmpl

```diff
@@ -19,7 +19,7 @@
 						{{end}}
 					{{else}}
 						{{if .NeedsPassword}}
-							<form class="ui form" action="/user/activate" method="post">
+							<form class="ui form" action="{{AppSubUrl}}/user/activate" method="post">
```

Comment:
Here is a nested form. Thats not allowed by the HTML specs.

Reply:
Hmm can we just remove the inner form?

Reply:
The inner form lacks the CSRF token and the outer form has no fields 🤷‍♂️

Reply:
@KN4CK3R should be block this issue because of this?

Reply:
The inner form was added in #13631. I would keep the form for this PR. While testing if it causes troubles I found some more bugs I will address in a new PR.

Reply:
I think it's fine just to remove the inner form

Reply:
#15473